contracts/contracts/coordination/InfractionCollector.sol

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+pragma solidity ^0.8.0;
+
+import "@openzeppelin-upgradeable/contracts/access/OwnableUpgradeable.sol";
+import "./Coordinator.sol";
+import "./TACoChildApplication.sol";
+
+contract InfractionCollector is OwnableUpgradeable {
+    Coordinator public coordinator;
+
+    // Reference to the TACoChildApplication contract
+    TACoChildApplication public tacoChildApplication;
```

Comment:
Minor thing, but would be a bit better to use `ITACoChildApplication` interface, it has `penalize` function

Reply:
Fixed in [Use interface for TACo Child Application in Infraction Collector](https://github.com/nucypher/nucypher-contracts/pull/267/commits/eea7b4a80ec964ee1f6a2d4d836fe65a3d24ec9e)